tools/test/h5repack/CMakeTests.cmake

```diff
@@ -1688,14 +1754,19 @@
   ADD_H5_TEST (HDFFV-7840 "TEST" h5diff_attr1.h5)
 
 # test CVE-2018-17432 fix
-  set (arg h5repack_CVE-2018-17432.h5 --low=1 --high=2 -f GZIP=8 -l dset1:CHUNK=5x6)
-  set (TESTTYPE "TEST")
-  ADD_H5_FILTER_TEST (HDFFV-10590 "" ${TESTTYPE} 1 ${arg})
+  set (arg --low=1 --high=2 -f GZIP=8 -l dset1:CHUNK=5x6)
+  add_test (
+      NAME H5REPACK-HDFFV-10590
+      COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:h5repack> ${arg} ${PROJECT_BINARY_DIR}/testfiles/h5repack_CVE-2018-17432.h5 ${PROJECT_BINARY_DIR}/testfiles/out-HDFFV-10590.h5repack_CVE-2018-17432.h5
+  )
+  set_tests_properties (H5REPACK-HDFFV-10590 PROPERTIES WILL_FAIL "true")
 
 # test CVE-2018-14460 fix
-  set (arg h5repack_CVE-2018-14460.h5)
-  set (TESTTYPE "TEST")
-  ADD_H5_FILTER_TEST (HDFFV-11223 "" ${TESTTYPE} 1 ${arg})
+  add_test (
+      NAME H5REPACK-HDFFV-11223
+      COMMAND ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:h5repack> ${PROJECT_BINARY_DIR}/testfiles/h5repack_CVE-2018-17432.h5 ${PROJECT_BINARY_DIR}/testfiles/out-HDFFV-11223.h5repack_CVE-2018-17432.h5
```

Comment:
Looks like this was copy-pasted to be the wrong file. Looks like this should be `${PROJECT_BINARY_DIR}/testfiles/h5repack_CVE-2018-14460.h5` and `${PROJECT_BINARY_DIR}/testfiles/out-HDFFV-11223.h5repack_CVE-2018-14460.h5`

Reply:
yes.